luigi/scheduler.py

```diff
@@ -150,6 +150,8 @@ class scheduler(Config):
 
     metrics_collector = parameter.EnumParameter(enum=MetricsCollectors, default=MetricsCollectors.default)
 
+    rerun_task_blocking_period = parameter.IntParameter(default=10)
```

Comment:
let's come up with a better name for this? I don't think we can come up with a name people will understand with a few words, but it should still make sense.

 * Maybe `stable_done_cooldown_secs`?
 * Whichever we choose we maybe want `_secs` in the name. There's a TimeDeltaParameter but I think we should avoid those fancy stuff in the scheduler. I dream that one day that the scheduler is written in another language even.

Also add docs to this parameter.